jpa/eclipselink.jpa.test/src/org/eclipse/persistence/testing/models/jpa/advanced/Material.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
```

Comment:
this is not valid header, it must contain EPL/EDL., see ie https://github.com/eclipse-ee4j/eclipselink/blob/master/project-admin/Sample.java for sample

Reply:
Changed the headers , anything Pls let me know